packages/react-aria-components/docs/examples/action-menu.mdx

```diff
@@ -47,22 +47,22 @@ function MenuExample() {
         <Button aria-label="Menu" className="inline-flex items-center justify-center rounded-md bg-black bg-opacity-20 bg-clip-padding border border-white/20 px-3 py-2 text-white hover:bg-opacity-30 pressed:bg-opacity-40 transition-colors cursor-default outline-none focus-visible:ring-2 focus-visible:ring-white/75"><RailIcon size="S" /></Button>
         <Popover className="p-1 w-56 overflow-auto rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 entering:animate-in entering:fade-in entering:zoom-in-95 exiting:animate-out exiting:fade-out exiting:zoom-out-95 fill-mode-forwards origin-top-left">
           <Menu className="outline-none">
-            <MenuItem id="new">New…</MenuItem>
-            <MenuItem id="open">Open…</MenuItem>
+            <Item id="new">New…</Item>
+            <Item id="open">Open…</Item>
             <Separator className="bg-gray-300 h-[1px] mx-3 my-1" />
-            <MenuItem id="save">Save</MenuItem>
-            <MenuItem id="save-as">Save as…</MenuItem>
+            <Item id="save">Save</Item>
+            <Item id="save-as">Save as…</Item>
             <Separator className="bg-gray-300 h-[1px] mx-3 my-1" />
-            <MenuItem id="print">Print…</MenuItem>
+            <Item id="print">Print…</Item>
           </Menu>
         </Popover>
       </MenuTrigger>
     </div>
   );
 }
 
-function MenuItem(props: ItemProps) {
-  return <Item {...props} className="group flex w-full items-center rounded-md px-3 py-2 box-border outline-none cursor-default text-gray-900 focus:bg-violet-500 focus:text-white" />;
+function Item(props: MenuItemProps) {
```

Comment:
possibly confusing to people who have been using the old style for a while, they may not realize that in this example they have been swapped. maybe call it `ActionItem` or something to just make it obvious? Like you did in status-select.mdx